### DISCUSSION TITLE
Emlak İhbar Sitesi

### DISCUSSION THREAD

**Discussion by coderserdar** [acikkaynak/deprem-yardim-projesi#98] [Ideas]
Merhaba arkadaşlar.
Hepimizin bildiği gibi deprem sonrasında pek çok ev yıkıldı ve depremzede vatandaşlarımız evsiz kaldı.
Binalar tekrardan yapılana kadar imkanı olanlar başka şehirden veya aynı şehirden daha sağlam ev almak, imkanı olmayanlar ise kiralık eve çıkmak durumunda kalacak.
Ancak maalesef bazı insanlarımız bu durumu kendi lehine fırsata çevirmek için
Satılık veya kiralık ilanlarında fahiş artırımlara gidiyorlar.
Örnek bir tanesini buraya ekran görüntüsü olarak ekliyorum.

![Screenshot_1](https://user-images.githubusercontent.com/11051271/218432093-c0cb06d6-4ccc-4f91-b1f3-02682a731f4e.png)

Bu gibi durumları engelleyebilmek için
Ufak bir web sayfası geliştirip devreye alabiliriz.
Sahibinden.com sitesinde bildiğim kadarıyla bir kullanıcının 200 adet ilanı favorilerine ekleme gibi bir imkan var.
Discord grubunda görev almaya çalışıp herhangi bir görev alamayan arkadaşlar eğer uygun görülür de site yapılırsa
Özellikle kiralık ilanlarda seçecekleri ilanları favorilerine ekleyip, fahiş artışları buraya girerek kayıt altına alınmasını sağlayabilirler.
Burada biriken bilgilerin topluca çevre şehircilik bakanlığına veya maliye bakanlığına ihbar edilmesi de mümkün olabilir.

Taslak olarak aşağıdaki gibi bir ekran hazırlanabilir.

![New Wireframe 1 copy](https://user-images.githubusercontent.com/11051271/218433765-d759bd03-4622-4b4c-96dc-4731e87ae7be.png)

Elbette bu işin bir hukuki boyutu vardır. Ancak yukarıdaki ekran görüntüsünde olduğu gibi bu fırsatçıların ifşa edilmesi gerektiğini düşünüyorum.
Sizlerden gelecek görüşlere göre de bu projenin kapsamı geliştirilebilir.
İyi günler diliyorum

**Comment by sercangoger** [comment]
bunları ifşa etmek gerçekten gerekli olmalı. herkesin herşeyi görmesi gerekiyor artık.
